src/scripts/main.js

```diff
@@ -1,3 +1,209 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+
+let table = document.querySelector('table');
+let headers = [...table.querySelectorAll('th')];
+let tbody = table.querySelector('tbody');
+let rows = [...tbody.querySelectorAll('tr')];
+
+const init = () => {
+  table = document.querySelector('table');
+  headers = [...table.querySelectorAll('th')];
+  tbody = table.querySelector('tbody');
+  rows = [...tbody.querySelectorAll('tr')];
+};
+
+const currencyFormat = (number) =>
+  '$' + new Intl.NumberFormat('en-US').format(number);
+
+// Add active class to row when clicked
+
+rows.forEach((row) => {
```

Comment:
The function `init` which initializes the table, headers, tbody, and rows is declared but it's not called at the start of the script. It's recommended to call this function at the start to ensure that all these variables are correctly initialized before they are used.